src/app/views/auth/auth.less

```diff
@@ -149,3 +149,272 @@
     height: 34px;
   }
 }
+
+input.fluroblue[type=radio] {
```

Comment:
`fluroblue` is color specific to maestrano, please use an agnostic selector name and remove those occurences.